### PR TITLE
Lesson routes - Seeds - Model Updates - Tests

### PR DESCRIPTION
**Pull request by Gabemb**
...sorry

### CODE REVIEW

File: server/web/db/models/user.js [hunk 1]
```diff
@@ -38,7 +39,7 @@ module.exports = (sequelize, DataTypes) => {
     classMethods: {
       associate(models) {
         // this column is for which class info should be displayed on the User's profile page.
-        User.belongsTo(models.class, { as: 'lastClassViewed' });
+        User.belongsTo(models.class, { foreignKey: 'lastClassViewed' });
```
Comment (samanthabretous) [inline server/web/db/models/user.js]: we have to use 'as' to include the associated model. It is fixed with pull #29

File: server/web/routes/class/class_middleware.js [hunk 2]
```diff
@@ -32,7 +32,22 @@ const getClassByEnrollmentCode = (req, res) => {
   .catch(() => res.sendStatus(500));
 };
 
+// /api/classes/info/:classId
+const fetchClassInfo = (req, res) => {
+  models.class.findById(req.params.classId, {
+    include: [{ all: true }],
```
Comment (samanthabretous) [inline server/web/routes/class/class_middleware.js]: do we need to include all associations with this route?

File: test/user_test.js [hunk 3]
```diff
@@ -1,26 +1,40 @@
...
+      lastName: 'Franklin',
+      email: 'kfranklin@msb.com',
+      username: 'kfranklin',
+      password: 'pass2',
+      position: 'Student',
+    },
+  ];
+
+  before(() => User.sync({ force: true })
+      .then(() => User.bulkCreate(users, { returning: true }))
```
Comment (samanthabretous) [inline test/user_test.js]: are we using the seed file or seeding in the test. Curious to know what `returning true` does

File: server/web/seed/index.js [hunk 4]
```diff
@@ -1,64 +1,87 @@
...
+    .then(() => {
+      models.assignment.bulkCreate(assignmentData);
+    })
+    .then(() => models.work.create(workData[0]))
+    .then((work) => {
+      work.addUser(1);
+      return models.work.create(workData[1]);
+    })
+    .then((work) => {
+      work.addUser(1);
```
Comment (samanthabretous) [inline server/web/seed/index.js]: is there a way we can divvy out the work to other users. Not 100% sure how the work is linked to the lessons, and what lessons are linked to the users.

File: test/user_test.js (hunk 3 above)
Comment (Gabemb) [inline test/user_test.js]: We're using the seed file for development.

File: server/web/routes/class/class_middleware.js (hunk 2 above)
Comment (Gabemb) [inline server/web/routes/class/class_middleware.js]: As long as we plan on populating a page with all the information relevant to that class, I'd say: yes. For now anyways.

**Pull request merged**